<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicsLineSeriesNode</class>
 <widget class="QWidget" name="GraphicsLineSeriesNode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(121, 121, 121,0);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>151</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(255, 255, 255);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="GraphicsDataArrayNode" name="YDataArray" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>15</y>
     <width>280</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>280</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>280</width>
     <height>120</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255, 255, 255, 0);

border:0px </string>
   </property>
  </widget>
  <widget class="GraphicsDataArrayNode" name="XDataArray" native="true">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>15</y>
     <width>280</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>280</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>280</width>
     <height>120</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255, 255, 255, 0);

border:0px </string>
   </property>
  </widget>
  <widget class="QLabel" name="lineSeriesType">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>110</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>120</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(255, 255, 255) ;
border-radius:5px;
background-color: rgba(221, 221, 221, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>线图序列</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="OutputPort" name="outputPort">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>70</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
							 border: 1px solid #333333;
							 padding: 4px;
							 border-radius:8px;

							 background-color: rgb(255, 255, 255);
						 }

						 QPushButton:hover {
							 background-color: rgb(52, 167, 255);
							 border-color: #444444;
						 }

						 QPushButton:pressed {
							 background-color: rgb(46, 39, 239);
							 border-color: #333333;
							 color: yellow;
						 }

						 QPushButton:disabled {
							 color: #333333;
						 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="InputPort" name="inputPort">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
							 border: 1px solid #333333;
							 padding: 4px;
							 border-radius:8px;

							 background-color: rgb(255, 255, 255);
						 }

						 QPushButton:hover {
							 background-color: rgb(52, 167, 255);
							 border-color: #444444;
						 }

						 QPushButton:pressed {
							 background-color: rgb(46, 39, 239);
							 border-color: #333333;
							 color: yellow;
						 }

						 QPushButton:disabled {
							 color: #333333;
						 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineSeriesName">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>0</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(255, 255, 255) ;
border-radius:5px;
background-color: rgba(221, 221, 221, 0);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InputPort</class>
   <extends>QPushButton</extends>
   <header location="global">NodeShow/GraphicsNode/GraphicsNode/InputPort.h</header>
  </customwidget>
  <customwidget>
   <class>OutputPort</class>
   <extends>QPushButton</extends>
   <header location="global">NodeShow/GraphicsNode/GraphicsNode/OutputPort.h</header>
  </customwidget>
  <customwidget>
   <class>GraphicsDataArrayNode</class>
   <extends>QWidget</extends>
   <header location="global">GraphicsDataArrayNode.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
